google-cloud-accessapproval/src/test/java/com/google/cloud/accessapproval/v1/AccessApprovalAdminClientTest.java

```diff
@@ -229,7 +421,9 @@ public void approveApprovalRequestExceptionTest() throws Exception {
   public void dismissApprovalRequestTest() throws Exception {
     ApprovalRequest expectedResponse =
         ApprovalRequest.newBuilder()
-            .setName("name3373707")
+            .setName(
+                ApprovalRequestName.ofProjectApprovalRequestName("[PROJECT]", "[APPROVAL_REQUEST]")
+                    .toString())
```

Comment:
These things are really what you want in your tests?  May be ok for mocks, but probably not for IT tests.